javascripts/custom-repos.js

```diff
@@ -64,9 +61,7 @@ var customRepos = [
 // Custom repo language, different than that defined by GitHub
 var customRepoLanguage = {
   'liquid': 'Liquid',
-  'Timber': 'Liquid',
```

Comment:
We can remove the associated .gif at the bottom of the file:
`'Timber': '/images/repo-avatars/timber.gif',`